deck-core/src/main/kotlin/com/deck/core/delegator/DeckEntityStrategizer.kt

```diff
@@ -2,76 +2,26 @@ package com.deck.core.delegator
 
 import com.deck.common.content.node.NodeGlobalStrategy
 import com.deck.common.entity.RawChannel
+import com.deck.common.entity.RawChannelCategory
 import com.deck.common.entity.RawPartialSentMessage
 import com.deck.common.entity.RawUser
 import com.deck.common.util.GenericId
-import com.deck.common.util.asNullable
 import com.deck.common.util.mapToBuiltin
 import com.deck.core.DeckClient
-import com.deck.core.entity.Channel
-import com.deck.core.entity.Message
-import com.deck.core.entity.SelfUser
-import com.deck.core.entity.User
+import com.deck.core.entity.*
 import com.deck.core.entity.impl.*
-import com.deck.core.entity.misc.forcefullyWrap
+import com.deck.gateway.entity.RawPartialTeamChannel
 import java.util.*
 
 public class DeckEntityStrategizer(private val client: DeckClient) : EntityStrategizer {
```

Comment:
I think for this class we may stay decoding this manually, since the focus of our entities classes are not to decode or map elements, but rather to provide simple and clean access to guilded properties.